com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerNotifier.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using MLAPI.Logging;
+
+namespace MLAPI.Profiling
+{
+    public static class ProfilerNotifier
```

Comment:
so obviously this PR introduces stuff more than just tests right?

Reply:
I agree, originally the PR was just tests but should probably update the commit message to refactor with the expanded scope